ush/parsing_model_configure_DATM.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+#####
+## "parsing_model_configure_DATM.sh"
+## This script writes model configure file
+## for DATM model
+##
+## This is the child script of ex-global forecast,
+## writing model configure file for DATM
+## This script is a direct execution.
+#####
+
+DATM_model_configure(){
+
+rm -f model_configure
+cat > model_configure <<EOF
+total_member:              ${ENS_NUM:-1}
+print_esmf:                ${print_esmf:-.true.}
+PE_MEMBER01:               $NTASKS_TOT
```

Comment:
I think this option is gone from the model.

Reply:
See my PR #402 has the options that are removed in it. You could look at that for which model_configure options are gone although this is for DATM, so you should probably look at the model_configure template for the DATM in ufs-weahter-model for the most up to date version.

Reply:
Do these changes work WITHOUT the hashtag change included in #402?

Reply:
OK These has been addressed also. I performed a quick test to ensure the model runs OK without total_member and PE_MEMBER01